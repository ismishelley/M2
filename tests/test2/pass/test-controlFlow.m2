int main(){
	float a;
	int b;

	a = 9.3;

	if (a > 5.1){
		printStr("If statement");
	}
	else{
		printStr("Else statement");
	}

	while (a > 8.0){
		a = a - 2.0;
		printStr("While loop");
	}

	for (b = 0; b < 2; ++b){
		printStr("For loop");
	}

	return 0;
}