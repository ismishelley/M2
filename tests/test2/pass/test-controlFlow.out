If statement
While loop
For loop
For loop
